docs/fcs/syntax-visitor.fsx

```diff
@@ -0,0 +1,185 @@
+﻿(**
+---
+title: Tutorial: SyntaxVisitorBase
+category: FSharp.Compiler.Service
+categoryindex: 300
+index: 301
+---
+*)
+(*** hide ***)
+#I "../../artifacts/bin/FSharp.Compiler.Service/Debug/netstandard2.0"
+(**
+Compiler Services: Using the SyntaxVisitorBase
+=========================================
+
+Syntax tree traversal is a common problem when interacting with the `FSharp.Compiler.Service`.  
```

Comment:
Well it's not a problem, it's <s>life</s> part of the process usually. Maybe `topic`?